README.md

```diff
@@ -23,7 +23,7 @@ We have organized the tutorials into three sections:
 Since quantum computing is new to most users, we want to find the best *Hello Quantum World* program and welcome submissions here. 
 
 ### 2. [Reference](reference/)<a id='reference'></a>
```

Comment:
Thanks. This looks good! But, do we need to say version 0.5.3? Because I think we are sure it will move forward quickly.

Reply:
I've changed all mentions of 0.5.3 to 0.5, dropping the minor version number.